### ISSUE TITLE
Feature request: seen-n-times?

### ISSUE THREAD

**Issue by madvas** [Day8/re-frame-async-flow-fx#3]
I'd find it useful if I could define that certain event would be fired after other event was fired n times. 

Possible syntax might looks somewhat like this:
```clojure
{:when :seen-n-times?
 :n 10       
 :events :something-loaded  
 :dispatch [:all-somethings-loaded]}}
```

What do you think of an idea? 
Thanks

**Comment by hipitihop** [comment]
@madvas, Apologies for our tardiness. At first blush I suspect this would cause some internal complexity and restructuring related to maintaining counters etc I'd like to better understand your use case. 

Typically when I need to load multiple things, I instrument each successful load to dispatch a unique event `:success-load-a` , `:success-load-b`, `:success-load-c` etc and then I setup the flow rule like this:

```clojure
{:when     :seen-all-of?
 :events   [:success-load-a :success-load-b :success-load-c]
 :dispatch [:all-somethings-loaded]}}
```

What do you think ? in your use case is there a reason this approach won't work ?

**Comment by madvas** [comment]
In my case it's 10 things that need to be loaded. I can do it the way you suggest, just seemed bit too laborious, so I thought I rather ask about this feature. If you feel like it would complect code bit too much, no big deal, we can close this.

**Comment by piotr-yuxuan** [comment]
I've been thrilled by your README, it's really awesome! I'm really fan of these ideas guys ;-) design sounds like it's very well crafted, I gonna read that educational code.

Reading this issue just popped a question for me: what about we have events vectors [here](https://github.com/Day8/re-frame-async-flow-fx/blob/master/src/day8/re_frame/async_flow_fx.cljs#L8):

``` clojure
(defn seen-all-of?
  [required-events seen-events]
  (empty? (set/difference required-events seen-events)))
```

Wouldn't it make super easy to write something like:
``` clojure
{:when     :seen-all-of?
 :events   (into [] (repeat 10 :success-load-a)
 :dispatch [:all-somethings-loaded]}}
```

**Comment by hipitihop** [comment]
@madvas Now that we have also merged #10/#20 in latest release, does your use case get easier ? Closing
